modules/cli/src/main/scala/scala/cli/commands/SharedOptions.scala

```diff
@@ -192,9 +192,18 @@ final case class SharedOptions(
 
   lazy val coursierCache = coursier.coursierCache(logging.logger.coursierLogger)
 
+  def programArgs(args: RemainingArgs) = args.unparsed
+
+  def inputArgs(args: RemainingArgs) = args.remaining
```

Comment:
NIT Do we really need those methods? They don't use any of the fields of `SharedOptions`, are single-line, and are only called twice. Couldn't they just be inlined?

Reply:
:+1: Done